### PR TITLE
Aavechan/pyusd listing

### PR DESCRIPTION
**Pull request by JosepBove**

### CODE REVIEW

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AddPYUSDToAaveV3EthereumMarket.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Motivation
+PYUSD is a USD-pegged stablecoin minted through Paxos. While it’s still young, it has grown to a circulating supply of over $230M with strong peg resilience. Onboarding this asset into Aave will:
+
+Build synergies between Aave and PYUSD.
+Offer Aave users an additional stablecoin option.
+Strengthen the relationship between the PYUSD & the GHO stablecoin. (Trident may explore the deployment of a PYUSD/GHO pool)
+We propose a PYUSD onboarding outside isolation mode but without collateral properties.
+
+If the community deems that PYUSD as collateral is immediately viable we stand ready to support that role on day one; otherwise following PYUSD maturity, the Aave governance can propose a collateral activation for PYUSD at a later stage.
```
Comment (marczeller) [inline src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AddPYUSDToAaveV3EthereumMarket.md:18]: this part is not needed

```suggestion
```

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AddPYUSDToAaveV3EthereumMarket.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| Base Variable Borrow Rate          |                                        0 % |
+| Variable Slope 1                   |                                        6 % |
+| Variable Slope 2                   |                                       80 % |
+| Uoptimal                           |                                       80 % |
+| Stable Borrowing                   |                                   DISABLED |
+| Stable Slope1                      |                                       13 % |
+| Stable Slope2                      |                                      300 % |
+| Base Stable Rate Offset            |                                        3 % |
+| Stable Rate Excess Offset          |                                        8 % |
+| Optimal Stable To Total Debt Ratio |                                       20 % |
```
Comment (sakulstra) [inline src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AddPYUSDToAaveV3EthereumMarket.md:50]:
```suggestion
```
would recommend remove stable from md as irrelevant

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AddPYUSDToAaveV3EthereumMarket.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Add PYUSD to Aave v3 Ethereum Market"
```
Comment (sakulstra) [inline src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AddPYUSDToAaveV3EthereumMarket.md:2]:
```suggestion
title: "Add PYUSD to Aave v3 Ethereum Pool"
```

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.t.sol [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 19084159);
+    proposal = new AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public {
+    defaultTest(
```
Comment (sakulstra) [inline src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.t.sol:28]: the diff was not committed

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: PYUSD,
+      assetSymbol: 'PYUSD',
+      priceFeed: 0x8f1dF6D7F2db73eECE86a18b4381F4707b918FB1,
+      eModeCategory: AaveV3EthereumEModes.NONE,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.DISABLED,
```
Comment (sakulstra) [inline src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol:41]: i see it's analog to snapshot, but why?

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    AaveV3Ethereum.POOL.supply(PYUSD, PYUSD_SEED_AMOUNT, address(AaveV3Ethereum.COLLECTOR), 0);
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: PYUSD,
+      assetSymbol: 'PYUSD',
+      priceFeed: 0x8f1dF6D7F2db73eECE86a18b4381F4707b918FB1,
```
Comment (sakulstra) [inline src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol:35]: the feed is still flagged as provisional on cl docs

File: lib/aave-helpers
Comment (sakulstra) [inline lib/aave-helpers:1]: why the update of this?

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol (hunk 6 above)
Comment (JosepBove) [reply src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol:35]: What is the way to go then?

Comment (marczeller) [reply src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol:35]: it's normal pyUSD is young but oracle is solid, we should not delay AIP by a month for this.

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol (hunk 5 above)
Comment (marczeller) [reply src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol:41]: new policy for alt stablecoins

File: src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol (hunk 6 above)
Comment (sakulstra) [reply src/20240125_AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket/AaveV3Ethereum_AddPYUSDToAaveV3EthereumMarket_20240125.sol:35]: we asked cl to check the data they have and evaluate the liquidity profile. We'll let you know once we hear back.

**Pull request merged**